From 487e2f25242c7d2cdae924eb29c8c5da6f96ff62 Mon Sep 17 00:00:00 2001
From: "Bernardo A. Rodrigues" <bernardoaraujor@gmail.com>
Date: Wed, 23 Oct 2019 20:28:02 +0200
Subject: [PATCH] flex trit encoding 3 trits per byte

---
 common/trinary/flex_trit.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/trinary/flex_trit.h b/common/trinary/flex_trit.h
index 27d951fc..478ebd78 100644
--- a/common/trinary/flex_trit.h
+++ b/common/trinary/flex_trit.h
@@ -14,7 +14,7 @@ extern "C" {
 
 #if !defined(FLEX_TRIT_ENCODING_1_TRIT_PER_BYTE) && !defined(FLEX_TRIT_ENCODING_3_TRITS_PER_BYTE) && \
     !defined(FLEX_TRIT_ENCODING_4_TRITS_PER_BYTE) && !defined(FLEX_TRIT_ENCODING_5_TRITS_PER_BYTE)
-#define FLEX_TRIT_ENCODING_1_TRIT_PER_BYTE
+#define FLEX_TRIT_ENCODING_3_TRITS_PER_BYTE
 #endif
 
 #include <stdbool.h>
-- 
2.17.1

