From 601b2580524b133d6e3c436906c7b6c9c749ef56 Mon Sep 17 00:00:00 2001
From: "Bernardo A. Rodrigues" <bernardoaraujor@gmail.com>
Date: Sat, 28 Sep 2019 21:19:27 +0200
Subject: [PATCH] add-linkopt-to-ciri-BUILD

---
 ciri/BUILD | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ciri/BUILD b/ciri/BUILD
index 62f8c94..823c574 100644
--- a/ciri/BUILD
+++ b/ciri/BUILD
@@ -49,6 +49,7 @@ cc_binary(
         "//utils/handles:rand",
         "//utils/handles:signal",
     ],
+    linkopts = ["-lm"],
 )
 
 cc_image(
-- 
2.7.4

