From e4eb0856df7b7c894c9fb49d4f5403c3a97ceef5 Mon Sep 17 00:00:00 2001
From: Bernardo Rodrigues <bernardoaraujor@gmail.com>
Date: Sun, 21 Jul 2019 11:20:13 +0200
Subject: [PATCH] add linkopt to ciri BUILD

---
 ciri/BUILD | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ciri/BUILD b/ciri/BUILD
index f86960b..8d5d156 100644
--- a/ciri/BUILD
+++ b/ciri/BUILD
@@ -47,6 +47,7 @@ cc_binary(
         "//utils/handles:rand",
         "//utils/handles:signal",
     ],
+    linkopts = ["-lm"],
 )
 
 cc_library(
-- 
2.7.4

