From 538d8c4d963675b425e42db415076b97e8fd76bf Mon Sep 17 00:00:00 2001
From: "Bernardo A. Rodrigues" <bernardoaraujor@gmail.com>
Date: Sat, 28 Sep 2019 21:46:08 +0200
Subject: [PATCH] remove unused definitions from WORKSPACE

---
 WORKSPACE | 43 -------------------------------------------
 1 file changed, 43 deletions(-)

diff --git a/WORKSPACE b/WORKSPACE
index a7000d3..eb43778 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -15,49 +15,6 @@ git_repository(
     remote = "https://github.com/iotaledger/toolchains.git",
 )
 
-android_sdk_repository(
-    name = "androidsdk",
-    api_level = 19,
-)
-
-android_ndk_repository(
-    name = "androidndk",
-    api_level = 19,
-)
-
-git_repository(
-    name = "build_bazel_rules_apple",
-    remote = "https://github.com/bazelbuild/rules_apple.git",
-    tag = "0.17.2",
-)
-
-load(
-    "@build_bazel_rules_apple//apple:repositories.bzl",
-    "apple_rules_dependencies",
-)
-
-apple_rules_dependencies()
-
-git_repository(
-    name = "io_bazel_rules_docker",
-    remote = "https://github.com/bazelbuild/rules_docker.git",
-    tag = "v0.8.1",
-)
-
-load(
-    "@io_bazel_rules_docker//repositories:repositories.bzl",
-    container_repositories = "repositories",
-)
-
-container_repositories()
-
-load(
-    "@io_bazel_rules_docker//cc:image.bzl",
-    _cc_image_repos = "repositories",
-)
-
-_cc_image_repos()
-
 load("@rules_iota//:defs.bzl", "iota_deps")
 load("//tools:snapshot.bzl", "fetch_snapshot_files")
 
-- 
2.7.4

