From 1d13ad7db458e9595b7610927e2052137817e5aa Mon Sep 17 00:00:00 2001
From: "Bernardo A. Rodrigues" <bernardoaraujor@gmail.com>
Date: Sat, 28 Sep 2019 22:11:53 +0200
Subject: [PATCH] set-iota_rules-to-fork

---
 WORKSPACE | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/WORKSPACE b/WORKSPACE
index eb43778..e7f1909 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -5,8 +5,8 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 git_repository(
     name = "rules_iota",
-    commit = "e08b0038f376d6c82b80f5283bb0a86648bb58dc",
-    remote = "https://github.com/iotaledger/rules_iota.git",
+    commit = "f6020f31e874d26b369fd94cd0a142fb0fe5b4f1",
+    remote = "https://github.com/bernardoaraujor/rules_iota.git",
 )
 
 git_repository(
-- 
2.7.4

