From 656fc6bd7ecf78e454c017889271d7d993c0eb15 Mon Sep 17 00:00:00 2001
From: "Bernardo A. Rodrigues" <bernardo.araujo@iota.org>
Date: Wed, 4 Mar 2020 17:43:10 +0100
Subject: [PATCH] add-YOCTO-option-to-CMakeLists

---
 CMakeLists.txt | 34 ++++++++++++++++++++--------------
 1 file changed, 20 insertions(+), 14 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 12dc3b9..fc48dd0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -18,6 +18,7 @@ endif()
 enable_language(C)
 enable_testing()
 
+option (YOCTO "Enable Building in Yocto" OFF)
 option(TRINARY_TEST "Trinary tests" OFF)
 
 #Trit Encoding flags
@@ -35,17 +36,20 @@ option(TE_5 "Trit encoding 5 trits per byte" OFF)
 
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
-include(ExternalProject)
 
-# libs in the sandbox
-#link_directories("${PROJECT_BINARY_DIR}/lib")
-link_directories("${CMAKE_INSTALL_PREFIX}/lib")
+if(NOT YOCTO)
+  include(ExternalProject)
 
-# external libs
-include(cmake/embear_logger.cmake)
-include(cmake/keccak.cmake)
-include(cmake/unity.cmake)
-include(cmake/uthash.cmake)
+  # libs in the sandbox
+  #link_directories("${PROJECT_BINARY_DIR}/lib")
+  link_directories("${CMAKE_INSTALL_PREFIX}/lib")
+
+  # external libs
+  include(cmake/embear_logger.cmake)
+  include(cmake/keccak.cmake)
+  include(cmake/unity.cmake)
+  include(cmake/uthash.cmake)
+endif()
 
 set(COMMON_TRINARY_DIR "common/trinary")
 set(COMMON_CRYPTO_DIR "common/crypto")
@@ -154,11 +158,13 @@ target_include_directories(common PUBLIC
   "${CMAKE_INSTALL_PREFIX}/include"
 )
 
-add_dependencies(common
-  ext_uthash
-  ext_keccak
-  ext_embear_logger
-)
+if(NOT YOCTO)
+  add_dependencies(common
+    ext_uthash
+    ext_keccak
+    ext_embear_logger
+  )
+endif()
 
 target_link_libraries(common PUBLIC 
   Threads::Threads
-- 
2.7.4

