From 17b09180b3c35e9b8967b9af72237b2ab6ea754f Mon Sep 17 00:00:00 2001
From: bernardo rodrigues <bernardo.araujo@silicon-gears.com>
Date: Fri, 9 Aug 2019 04:52:24 -0400
Subject: [PATCH] fix mam_msg_pubkey_e enum case

---
 iota/send-header.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/iota/send-header.c b/iota/send-header.c
index 7d8b116..6eec4a8 100644
--- a/iota/send-header.c
+++ b/iota/send-header.c
@@ -62,12 +62,12 @@ int main(int ac, char **av) {
     // Writing header to bundle
     trit_t msg_id[MAM_MSG_ID_SIZE];
 
-    if (msg_pubkey == mam_msg_pubkey_chid) {
+    if (msg_pubkey == MAM_MSG_PUBKEY_CHID) {
       if ((ret = mam_example_write_header_on_channel(&api, channel_id, bundle, msg_id)) != RC_OK) {
         fprintf(stderr, "mam_example_write_header failed with err %d\n", ret);
         return EXIT_FAILURE;
       }
-    } else if (msg_pubkey == mam_msg_pubkey_epid) {
+    } else if (msg_pubkey == MAM_MSG_PUBKEY_EPID) {
       tryte_t new_endpoint_id[MAM_ENDPOINT_ID_TRYTE_SIZE];
 
       if ((ret = mam_example_announce_new_endpoint(&api, channel_id, bundle, msg_id, new_endpoint_id)) != RC_OK) {
@@ -87,7 +87,7 @@ int main(int ac, char **av) {
         return EXIT_FAILURE;
       }
       return 0;
-    } else if (msg_pubkey == mam_msg_pubkey_epid1) {
+    } else if (msg_pubkey == MAM_MSG_PUBKEY_EPID1) {
       tryte_t new_endpoint_id[MAM_ENDPOINT_ID_TRYTE_SIZE];
 
       bundle_transactions_free(&bundle);
@@ -96,7 +96,7 @@ int main(int ac, char **av) {
         fprintf(stderr, "mam_example_announce_new_endpoint failed with err %d\n", ret);
         return EXIT_FAILURE;
       }
-    } else if (msg_pubkey == mam_msg_pubkey_chid1) {
+    } else if (msg_pubkey == MAM_MSG_PUBKEY_CHID1) {
       tryte_t new_channel_id[MAM_CHANNEL_ID_TRYTE_SIZE];
 
       if ((ret = mam_example_announce_new_channel(&api, channel_id, bundle, msg_id, new_channel_id)) != RC_OK) {
-- 
2.7.4

