From b68e042ff934f7ac8e3d23b59532a033760a0f41 Mon Sep 17 00:00:00 2001
From: bernardo rodrigues <bernardo.araujo@silicon-gears.com>
Date: Thu, 8 Aug 2019 11:20:24 -0400
Subject: [PATCH] fix mam_api_channel_create

---
 iota/common.c      | 2 +-
 iota/send-common.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/iota/common.c b/iota/common.c
index aa97bcc..dcec268 100644
--- a/iota/common.c
+++ b/iota/common.c
@@ -28,7 +28,7 @@ mam_psk_t const psk = {
 
 retcode_t mam_example_create_channel(mam_api_t *const api, tryte_t *const channel_id) {
   if (mam_channel_t_set_size(api->channels) == 0) {
-    mam_api_create_channel(api, TEST_MSS_DEPTH, channel_id);
+    mam_api_channel_create(api, TEST_MSS_DEPTH, channel_id);
   } else {
     mam_channel_t *channel = &api->channels->value;
     trits_to_trytes(trits_begin(mam_channel_id(channel)), channel_id, NUM_TRITS_ADDRESS);
diff --git a/iota/send-common.c b/iota/send-common.c
index 6b1809e..904fb9f 100644
--- a/iota/send-common.c
+++ b/iota/send-common.c
@@ -25,7 +25,7 @@ retcode_t mam_example_announce_new_channel(mam_api_t* const api, tryte_t const*
     }
   }
 
-  ERR_BIND_RETURN(mam_api_create_channel(api, MAM_EXAMPLES_MSS_HEIGHT, new_channel_id), ret);
+  ERR_BIND_RETURN(mam_api_channel_create(api, MAM_EXAMPLES_MSS_HEIGHT, new_channel_id), ret);
 
   if ((ret = mam_api_bundle_announce_new_channel(api, channel_id, new_channel_id, psks, NULL, 0, bundle, msg_id)) !=
       RC_OK) {
-- 
2.7.4

