From 9f5e620a006be6fe98233ae4b78b4ae95d995caf Mon Sep 17 00:00:00 2001
From: bernardo rodrigues <bernardo.araujo@silicon-gears.com>
Date: Thu, 8 Aug 2019 11:52:31 -0400
Subject: [PATCH] fix mam_api_bundle_write_header_on_endpoint

---
 iota/send-common.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/iota/send-common.c b/iota/send-common.c
index 904fb9f..e3f9f00 100644
--- a/iota/send-common.c
+++ b/iota/send-common.c
@@ -126,7 +126,7 @@ retcode_t mam_example_write_header_on_endpoint(mam_api_t* const api, tryte_t con
     }
   }
 
-  if ((ret = mam_api_bundle_write_header_on_endpoint(api, channel_id, endpoint_id, psks, NULL, 0, bundle, msg_id)) !=
+  if ((ret = mam_api_bundle_write_header_on_endpoint(api, channel_id, endpoint_id, psks, NULL, bundle, msg_id)) !=
       RC_OK) {
     return ret;
   }
-- 
2.7.4

