From 3173ac7f32f658ef12d56d32b25732928a3ddc90 Mon Sep 17 00:00:00 2001
From: bernardo araujo rodrigues <bernardo.araujo@silicon-gears.com>
Date: Mon, 29 Jul 2019 16:39:15 +0200
Subject: [PATCH] add lm linkopt

---
 app/BUILD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/BUILD b/app/BUILD
index 90d39bc..2e261e0 100644
--- a/app/BUILD
+++ b/app/BUILD
@@ -22,5 +22,5 @@ cc_binary(
         "//jsmn:jsmn",
     ],
     copts = ["-DLOGGER_ENABLE"],
-    linkopts = ["-pthread"],
+    linkopts = ["-pthread", "-lm"],
 )
-- 
2.7.4

