From 89e36a65aeb0a0c414885544cdde6b7a7968afbe Mon Sep 17 00:00:00 2001
From: "Bernardo A. Rodrigues" <bernardoaraujor@gmail.com>
Date: Sat, 28 Sep 2019 21:56:21 +0200
Subject: [PATCH] rm docker rules from ciri/BUILD

---
 ciri/BUILD | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/ciri/BUILD b/ciri/BUILD
index 823c574..8d5d156 100644
--- a/ciri/BUILD
+++ b/ciri/BUILD
@@ -9,8 +9,6 @@ config_setting(
     visibility = ["//visibility:public"],
 )
 
-load("@io_bazel_rules_docker//cc:image.bzl", "cc_image")
-load("@io_bazel_rules_docker//container:container.bzl", "container_image")
 load(":conf.bzl", "CIRI_MAINNET_VARIABLES")
 load(":conf.bzl", "CIRI_TESTNET_VARIABLES")
 
@@ -52,20 +50,6 @@ cc_binary(
     linkopts = ["-lm"],
 )
 
-cc_image(
-    name = "ciri_base_image",
-    binary = ":ciri",
-)
-
-container_image(
-    name = "ciri_image",
-    base = ":ciri_base_image",
-    mode = "755",
-    repository = "iotaledger",
-    stamp = True,
-    volumes = ["/app/ciri/ciri.runfiles/org_iota_entangled/ciri/db"],
-)
-
 cc_library(
     name = "core_shared",
     hdrs = ["core.h"],
-- 
2.7.4

