From 5942770391775eee19054441dbd7f3c9e7efe078 Mon Sep 17 00:00:00 2001
From: Bernardo Rodrigues <bernardoaraujor@gmail.com>
Date: Sun, 21 Jul 2019 11:23:43 +0200
Subject: [PATCH] set iota_rules to fork

---
 WORKSPACE | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/WORKSPACE b/WORKSPACE
index c59f918..6083cc5 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -4,8 +4,8 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 git_repository(
     name = "rules_iota",
-    commit = "73f598ad1ce3ba79ff22d747f723d6d5cbf351e1",
-    remote = "https://github.com/iotaledger/rules_iota.git",
+    commit = "a272523d58fbf4ad56f0f67d8875a10b61e54a4a",
+    remote = "https://github.com/bernardoaraujor/rules_iota.git",
 )
 
 git_repository(
-- 
2.7.4

